CHANGELOG.md

```diff
@@ -3,17 +3,17 @@
 ## [v3.2.1.dev0]
 
 ### Added
-- [Core] Added a mechanism to automatically retry failed tasks by @tomwhite
```

Comment:
Any reason you removed my name?

Reply:
No, I was planing to put names on the release, just to avoid putting my name everywhere :)

Reply:
I don't mind either way. Maybe do it in a separate PR which explains the motivation?

Reply:
BTW you can get GH to generate release notes which automatically adds contributors' names. https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#about-automatically-generated-release-notes